src/WebJobs.Script/Diagnostics/Extensions/LoggerExtension.cs

```diff
@@ -185,6 +191,11 @@ public static void ScriptStartUpLoadingStartUpExtension(this ILogger logger, str
             _scriptStartUpLoadingStartUpExtension(logger, startupExtensionName, null);
         }
 
+        public static void ScriptStartUpLoadedStartUpExtension(this ILogger logger, string startupExtensionName, string startupExtensionVersion)
```

Comment:
One more tiny nit: remove the second `StartUp` from here so it's `ScriptStartUpLoadedExtension`. Same with the private member... `_scriptStartUpLoadedExtension`